### PR TITLE
chore(ci): add fastlane ios to travis

### PR DESCRIPTION
**Pull request by lessp**
**Ticket:** [#187](https://trello.com/c/8II0jkq7/187-fastlane-build-in-travis-ios)
**Intent:**
**How to test (optional):**

### All of the following commands should pass.

- [ ] `npm test`
- [ ] `npm run lint`

### CODE REVIEW

File: ios/Egendata/Info.plist [hunk 1]
```diff
@@ -34,7 +34,7 @@
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>45</string>
+	<string>56</string>
```
Comment (raduachim) [inline ios/Egendata/Info.plist]: was this autogenerated by test runs? 

Do we need to set this to 0 when we merge or this is correct version matching what is in Testflight?

File: ios/Podfile [hunk 2]
```diff
@@ -25,7 +25,7 @@ target 'Egendata' do
   pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/glog.podspec'
   pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'
   # Add new pods below this line
-  pod 'ReactNativePermissions', :path => '../node_modules/react-native-permissions'
```
Comment (raduachim) [inline ios/Podfile]: was this removal intended?

Comment (lessp) [reply ios/Podfile]: Yep!

File: ios/Egendata/Info.plist (hunk 1 above)
Comment (raduachim) [reply ios/Egendata/Info.plist]: So let's try it live? 😄
